moved GTF parsing to external gtftools library

iskandr opened the pull request:
<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/hammerlab/pyensembl/127)
<!-- Reviewable:end -->

tavinathanson commented:
Looks good to me

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---

<sup>**[requirements.txt, line 11 \[r1\]](https://reviewable.io:443/reviews/hammerlab/pyensembl/127#-Jzn9J1z1YCQYncbFy0D)** ([raw file](https://github.com/hammerlab/pyensembl/blob/4b307d357ece202633ea69fb41a13fa426bf4fa3/requirements.txt#L11)):</sup>
Version?

---

<sup>**[setup.py, line 72 \[r1\]](https://reviewable.io:443/reviews/hammerlab/pyensembl/127#-Jzn9HrfMc80Mwnb5qIZ)** ([raw file](https://github.com/hammerlab/pyensembl/blob/4b307d357ece202633ea69fb41a13fa426bf4fa3/setup.py#L72)):</sup>
Version?

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/hammerlab/pyensembl/127)
<!-- Sent from Reviewable.io -->

iskandr commented:
Review status: 1 of 6 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---

<sup>**[requirements.txt, line 11 \[r1\]](https://reviewable.io:443/reviews/hammerlab/pyensembl/127#-Jzn9J1z1YCQYncbFy0D)** ([raw file](https://github.com/hammerlab/pyensembl/blob/4b307d357ece202633ea69fb41a13fa426bf4fa3/requirements.txt#L11)):</sup>
Added

---

<sup>**[setup.py, line 72 \[r1\]](https://reviewable.io:443/reviews/hammerlab/pyensembl/127#-Jzn9HrfMc80Mwnb5qIZ)** ([raw file](https://github.com/hammerlab/pyensembl/blob/4b307d357ece202633ea69fb41a13fa426bf4fa3/setup.py#L72)):</sup>
Added

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/hammerlab/pyensembl/127)
<!-- Sent from Reviewable.io -->

tavinathanson commented:
Good to merge

---

Reviewed 4 of 5 files at r2.
Review status: 5 of 6 files reviewed at latest revision, all discussions resolved, some commit checks broke.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/hammerlab/pyensembl/127)
<!-- Sent from Reviewable.io -->
